app/renderer/actions/Session.js

```diff
@@ -201,7 +201,10 @@ export function removeCapability (index) {
 
 function _addVendorPrefixes (caps, dispatch, getState) {
   const prefixedCaps = addVendorPrefixes(caps);
```

Comment:
you can just do `const {server, serverType, capsUUID} = getState().session` to get a short reference to each of these and avoid so many chars.